packages/@aws-cdk/integ-tests-alpha/lib/assertions/providers/lambda-handler/sdk.ts

```diff
@@ -97,7 +97,7 @@ export class AwsApiCallHandler extends CustomResourceHandler<AwsApiCallRequest,
     };
 
     let resp: AwsApiCallResult | { [key: string]: string } = respond;
-    if (request.outputPaths) {
+    if (request.outputPaths?.length) {
```

Comment:
The original behavior is the same for AwsCustomResource, and I think it would be more intuitive to make the properties passed to Lambda optional, what do you think?
https://github.com/aws/aws-cdk/blob/main/packages/aws-cdk-lib/custom-resources/lib/aws-custom-resource/runtime/aws-sdk-v3-handler/index.ts#L138-L142

Examples:
```diff
-  protected outputPaths: string[] = [];
+  protected outputPaths?: string[];
```
https://github.com/aws/aws-cdk/blob/main/packages/%40aws-cdk/integ-tests-alpha/lib/assertions/api-call-base.ts#L128

```diff
  public getAtt(attributeName: string): Reference {
-    this.outputPaths.push(attributeName);
+    (this.outputPaths ??= []).push(attributeName);
    this.flattenResponse = 'true';
    return this.apiCallResource.getAtt(`apiCallResponse.${attributeName}`);
  }
  ```
  https://github.com/aws/aws-cdk/blob/main/packages/%40aws-cdk/integ-tests-alpha/lib/assertions/api-call-base.ts#L137-L141